packages/http-client-csharp/generator/TestProjects/CadlRanch.Tests/Http/Resiliency/SrvDriven/V2/SrvDrivenV2Tests.cs

```diff
@@ -1,85 +1,85 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-using NUnit.Framework;
-using System.Threading.Tasks;
-using Resiliency.SrvDriven.V2;
+// using NUnit.Framework;
+// using System.Threading.Tasks;
+// using Resiliency.SrvDriven.V2;
 
 namespace TestProjects.CadlRanch.Tests.Http.Resiliency.SrvDriven.V2
 {
-    public partial class SrvDrivenV2Tests : CadlRanchTestBase
```

Comment:
Need to retain these tests.

Reply:
Here is the issue: https://github.com/microsoft/typespec/issues/5567

Reply:
Fixed